src/components/mainComponent/MainComponent.spec.js

```diff
@@ -10,7 +11,7 @@ describe('Main Component', () => {
   it('renders a new page after start button is click', () => {
     const wrapper = mount(<Main />);
 
-    wrapper.find('.human-page').simulate('click');
+    wrapper.find(ButtonComponent).at(0).simulate('click');
```

Comment:
From our conversation yesterday, remember that you need a `wrapper.update` after a click.

Reply:
To elaborate, you need a `wrapper.update()` after _each_ click.